### ISSUE TITLE
Ocassional Crash after running skipfish

### ISSUE THREAD

**Issue by delta24** [owtf/owtf#435]
@7a :  

```bash
[+] This was a great day for science!


[-] 
The Command was aborted by the user: Please check the report and plugin output files
[-] Time=21h,  3m, 37s, 921ms
[!] Exception during reset or similar
Traceback (most recent call last):
  File "/usr/local/lib/python2.7/dist-packages/sqlalchemy/pool.py", line 565, in _finalize_fairy
    fairy._reset(pool, echo)
  File "/usr/local/lib/python2.7/dist-packages/sqlalchemy/pool.py", line 699, in _reset
    pool._dialect.do_rollback(self)
  File "/usr/local/lib/python2.7/dist-packages/sqlalchemy/engine/default.py", line 406, in do_rollback
    dbapi_connection.rollback()
OperationalError: SSL SYSCALL error: EOF detected
```

**Comment by delta24** [comment]
Main issue here: https://bitbucket.org/zzzeek/sqlalchemy/issues/3021/ssl-eof-not-detected-as-disconnect-in

**Comment by delta24** [comment]
Fixed in 128a91ddfa2611e8e0fbf0eeed64093d24c4fc83, 7f9cae9be4529b07169881c4bdf42c8e523ab3a2 and 44c5df75a9fe9d6eeb2eb5ce15cfecb24ff2845f.
